docs/user/guides/canonical-urls.rst

```diff
@@ -26,7 +26,7 @@ and letting Read the Docs define it.
 MkDocs
 ~~~~~~
 
-For :doc:`MkDocs </intro/getting-started-with-mkdocs>` we do not define your canonical domain automatically,
+For :doc:`MkDocs </intro/mkdocs>` we do not define your canonical domain automatically,
```

Comment:
We don't define the canonical domain automatically on any tool now. If there are documentation saying that, we should change it and point to the specific guide (the ones we are writing here).

Reply:
I was super confused by this for a moment. Right, I only updated the file for the broken link. Probably I should fix the merge resolution first. So thanks for the pointer on that update.